server/src/test/java/org/elasticsearch/repositories/blobstore/BlobStoreRepositoryCleanupIT.java

```diff
@@ -107,4 +111,40 @@ private String startBlockedCleanup(String repoName) throws Exception {
         waitForBlock(masterNode, repoName, TimeValue.timeValueSeconds(60));
         return masterNode;
     }
+
+    public void testCleanupOldIndexN() throws ExecutionException, InterruptedException {
+        internalCluster().startNodes(Settings.EMPTY);
+
+        final String repoName = "test-repo";
+        logger.info("-->  creating repository");
```

Comment:
I'm not sure such info logs are very helpful

Reply:
I gotta disagree here :) I find these super helpful when it comes to debugging test failures from uncaught exceptions (esp. failing asserts on some non-test thread) where without these you often have a very hard time figuring out where exactly you were in the test at the time of the failure.

I wonder if we could do a better job on the loggin gside though ... there's not much point actually logging this stuff to the output on passing tests (basically like we have on the CLI). We could bring that up with core/infra?